test/user_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+
+class UserTest < Minitest::Test
+  def test_user_exists
+    sal = User.new("Sal")
+    assert_instance_of User, sal
+  end
+
+  def test_can_learn_joke
+    sal = User.new("Sal")
+    joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+    sal.learn(joke_1)
+    sal.learn(joke_2)
+    assert_equal sal.jokes.length, 2
```

Comment:
same thing here-- asserting length isn't super helpful.  Assert that it's an array with joke_1 and joke_2.